js/src/components/paid-ads/billing-card/billing-setup-card.js

```diff
@@ -58,23 +73,28 @@ const BillingSetupCard = ( { billingUrl, onSetupComplete } ) => {
 					<div className="gla-google-ads-billing-setup-card__description__helper">
 						{ createInterpolateElement(
 							__(
-								'You will be directed to Google Ads for this step. In case your browser is unable to open the pop-up, <link>click here instead</link>.',
+								'You will be directed to Google Ads for this step. In case your browser is unable to open the pop-up, <link>click here instead <icon /></link>.',
 								'google-listings-and-ads'
 							),
-							{ link: <ExternalLink href={ billingUrl } /> }
+							{
+								// compatibility-code "WP < 6.2" -- The following two components can be replaced with `ExternalLink` as it started supporting onClick since WordPress 6.2
+								link: (
+									// eslint-disable-next-line jsx-a11y/anchor-has-content
+									<a
+										target="_blank"
+										rel="external noreferrer noopener"
+										href={ billingUrl }
+										onClick={ handleClick }
+									/>
+								),
+								icon: (
+									<Icon icon={ externalIcon } size={ 12 } />
+								),
+							}
 						) }
 					</div>
 				</div>
-				<AppButton
-					isSecondary
-					onClick={ handleClick }
-					eventName="gla_ads_set_up_billing_click"
-					eventProps={ {
-						context: 'setup-ads',
```

Comment:
It used to set the context as `setup-ads`, but right now it showed `setup-mc` when I clicked `Set up billing` button. Is it expected?

<img width="657" alt="Screenshot 2024-06-03 at 16 04 51" src="https://github.com/woocommerce/google-listings-and-ads/assets/914406/b8da15b9-e35a-46d5-87cd-2a0965826d45">

Reply:
Yes, `'setup-mc'` is expected if you were doing extension onboarding. The billing setup card is also used in the onboarding flow, where the value of context is expected to be `'setup-ads'`.

It's true that there's some inconsistency and confusion in the use of these context values, so I've put them down in tracks.js momentarily.

https://github.com/woocommerce/google-listings-and-ads/blob/a63df8d70c682c380a3db39378fa86de92e9f365/js/src/utils/tracks.js#L27-L40